packages/dai-plugin-mcd/test/SystemDataService.spec.js

```diff
@@ -9,7 +9,8 @@ beforeAll(async () => {
   service = maker.service(ServiceRoles.SYSTEM_DATA);
 });
 
-test('get annual base rate', async () => {
+// This will fail until the next release (>v0.2.9)
+xtest('get annual base rate', async () => {
```

Comment:
@levity Are you suggesting an update to the contracts in the testchain to include Gonzalo's fix? I can do this, I just didn't because it's not part of the 0.2.9 release.

Reply:
Was the issue with a contract or just a configuration value? I thought it was the latter, in which case we would just need to tweak a json file before building the snapshot, without having to check in any changes to the deploy scripts repo